course_catalog/tasks.py

```diff
@@ -34,8 +34,27 @@ def get_mitx_data():
 
 
 @app.task(acks_late=True)
+def get_ocw_courses_with_retry(
```

Comment:
minor nitpick: this could be renamed to `get_ocw_course_with_retry` since it only affects 1 course at a time.